src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -578,7 +578,14 @@ namespace DataHeatBalance {
         int ZoneEqNum;                            // Controlled zone equip config number
         int PlenumCondNum;                        // Supply or return plenum conditions number, 0 if this is not a plenum zone
         int TempControlledZoneIndex;              // this is the index number for TempControlledZone structure for lookup
-        //            Pointers to Surface Data Structure
+        // Pointers to Surface Data Structure
+        // AllSurfF                                                                   AllSurfL
```

Comment:
This is very helpful, but I would pay close attention to the alignment and possibly add `|` characters to draw vertical lines to show the groupings. 
Also, some of these will equal each other, e.g. `HTSurfaceFirst = OpaqOrIntMassSurfaceFirst`. 
And comments explaining how these are set if a particular class of surfaces are not present in the zone. e.g. if there are no domes, what are `TDDDomeFirst and TDDDomeLast` set to?